src/app/stationform/stationform.component.ts

```diff
@@ -0,0 +1,23 @@
+import {Component, NgModule, OnInit} from '@angular/core';
+import {GeoJsonData} from '../models/geojsondata';
+import {GeoJsonDataService} from '../services/geo-json-data.service';
+
+@Component({
+	selector: 'diloc-stationform',
+  	templateUrl: './stationform.component.html',
+  	styleUrls: ['./stationform.component.scss']
+})
+
+export class StationformComponent implements OnInit {
+
+	public geoJsonData: GeoJsonDataService;
+
+	public newGeoJson: GeoJsonData = { stationName: "placeholder", lat: 0, lon:0};
```

Comment:
The placeholder can be set inside the form. lat and lon should be null (if possible) that the system knows that there has to be added something.
```suggestion
	public newGeoJson: GeoJsonData = { stationName: "", lat: null, lon: null};
```